Package.swift

```diff
@@ -0,0 +1,43 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Multiplay",
+    platforms: [
+        .macOS(.v10_15),
+        .iOS(.v11)
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Multiplay",
+            targets: ["Multiplay"]),
+        .executable(
+            name: "MultiplayChat",
+            targets: ["MultiplayChat"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/SwiftyJSON/SwiftyJSON.git", from: "5.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
```

Comment:
perhaps a src/client and src/server split could be good in the future. A pure Swift server may be a good idea at some point.

Reply:
https://github.com/jacques-gasselin/multiplay/projects/1#card-52056173